packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -5923,6 +5928,19 @@ class _Editable extends MultiChildRenderObjectWidget {
   }
 }
 
+class _NeverUserScrollableScrollPhysics extends ScrollPhysics {
+  /// Creates scroll physics that does not let the user scroll, but allows for programatic scrolling.
```

Comment:
Nit: Maybe say that it does not let the user scroll "by dragging" or "by panning" to be more specific? Or maybe I'm being too pedantic.